sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamExecution.scala

```diff
@@ -331,7 +338,6 @@ class StreamExecution(
 
     val batchTime = (System.nanoTime() - startTime).toDouble / 1000000
     logInfo(s"Completed up to $availableOffsets in ${batchTime}ms")
```

Comment:
There is a log in `constructNextBatch` when `dataAvailable` is true: `logInfo(s"Committed offsets for batch $currentBatchId.")`. And I think we should not log anything when `dataAvailable` is false. Otherwise, when we run the batches as fast as possible, there will be too many meaningless logs.